reactioncommerce/api-plugin-shops
# feat: move files to plugin

This plugin replaces the internal `shop` service. 

Once this PR is merged and this plugin is published to `npm`, there will be a related PR in `reaction` to swap out this plugin for the existing version.

## Testing

- see that the plugin is loaded upon startup in Reaction (use the `systemInformation` query)
- use the shop queries and mutations through the GraphQL API and verify they are working

Example test queries/mutations:
```
query {
  primaryShopId
}

query {
  primaryShop {
    _id
    name
    slug
  }
}

query {
  shopBySlug(slug: "set-shop-slug"){
    _id
    name
    slug
    description
  }
}

mutation {
  updateShop(input: {description: "new shop description", shopId: "set-shop-id"}){
    shop {
      _id
      name
      description
      slug
    }
  }
}
```

Issue https://github.com/reactioncommerce/reaction/issues/6180

Signed-off-by: Maria Paktiti <maria.paktiti@gmail.com>
